PendingReleaseNotes

```diff
@@ -3,3 +3,501 @@
 * RGW's default backend for `rgw_enable_ops_log` changed from RADOS to file.
   The default value of `rgw_ops_log_rados` is now false, and `rgw_ops_log_file_path`
   defaults to "/var/log/ceph/ops-log-$cluster-$name.log".
+* It is not an officially support feature while using SPDK backend (e.g.,
+  NVMe-oF tcp target) in Bluestore.
```

Comment:
```suggestion
* SPDK backend for Bluestore is now able to connect to NVMe-OF TCP target. Please note, this is not an officially support feature.
```

Reply:
The SPDK back end for BlueStore is now able to connect to an NVMeoF target.  Please note that this is not an officially supported feature.

Reply:
ack